<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>noise_marking_gui</class>
 <widget class="QDialog" name="noise_marking_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PPG/ECG Noise Marker</string>
  </property>
  <layout class="QHBoxLayout" name="rootLayout" stretch="0,1">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="leftPanel">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <layout class="QGridLayout" name="window_length_settings">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="window_length_label">
         <property name="text">
          <string>Window Length</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rb_10s">
         <property name="text">
          <string>10s</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="rb_30s">
         <property name="text">
          <string>30s</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="rb_1m">
         <property name="text">
          <string>1m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="rb_5m">
         <property name="text">
          <string>5m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="rb_10m">
         <property name="text">
          <string>10m</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="eight_hour_nav">
       <item>
        <widget class="QPushButton" name="prev8hours">
         <property name="text">
          <string>Prev 8h</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next8hours">
         <property name="text">
          <string>Next 8h</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="undo_clearall">
       <item>
        <widget class="QPushButton" name="undo_button">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearall_button">
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="skip_and_save">
       <item>
        <widget class="QPushButton" name="skip_button">
         <property name="text">
          <string>Skip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="finalize_button">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ecg_marks">
       <item>
        <widget class="QPushButton" name="startNoiseECG">
         <property name="text">
          <string>Mark ECG Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopNoiseECG">
         <property name="text">
          <string>Mark ECG End</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ppg_marks_row">
       <item>
        <widget class="QPushButton" name="startNoisePPG">
         <property name="text">
          <string>Mark PPG Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopNoisePPG">
         <property name="text">
          <string>Mark PPG End</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="typeRow">
       <item>
        <widget class="QLabel" name="marking_type_label">
         <property name="text">
          <string>Mark Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="marking_type">
         <item>
          <property name="text">
           <string>Noise/Artifact</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Conduction Delay</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SVT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PVC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PAC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Benign Arrhythmia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Significant Arrhythmia</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="scrollRow">
       <item>
        <widget class="QLabel" name="seconds_to_scroll_label">
         <property name="text">
          <string>Scroll Dist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="skip_interval_box">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>5.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="marking_type_label_2">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Marking Type Legend</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="marking_type_legend">
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="6" column="0" colspan="2">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="signif_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: rgb(0, 255, 255); border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="signif_desc">
           <property name="text">
            <string>Significant Arrhythmia</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="benign_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: rgb(255, 128, 255); border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="benign_desc">
           <property name="text">
            <string>Benign Arrhythmia</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="pvc_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: rgb(128, 255, 0); border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pvc_desc">
           <property name="text">
            <string>PVC</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="noise_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: yellow;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noise_desc">
           <property name="text">
            <string>Noise/Artifact</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="delay_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: rgb(128,0,128); border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="delay_desc">
           <property name="text">
            <string>Conduction Delay</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="vt_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: green; border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vt_desc">
           <property name="text">
            <string>VT</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="pac_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: rgb(255,128, 0); border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pac_desc">
           <property name="text">
            <string>PAC</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="svt_color">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: blue; border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="svt_desc">
           <property name="text">
            <string>SVT</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="af_color">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>background-color: red; border-radius:3px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="af_desc">
             <property name="text">
              <string>AF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="sleep_states_label">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sleep States Legend</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="sleep_state_legend">
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="2" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="n3_c">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: red; border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="n3_l">
           <property name="text">
            <string>NREM3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="n2_c">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: cyan; border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="n2_l">
           <property name="text">
            <string>NREM2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="n1_c">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: blue; border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="n1_l">
           <property name="text">
            <string>NREM1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="rem_c">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: green; border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rem_l">
           <property name="text">
            <string>REM</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="wake_c">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>background-color: black; border-radius:3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="wake_l">
           <property name="text">
            <string>Wake</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="vspacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="plotArea" stretch="0,1,2,2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="topLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ampogram_and_sleepstates">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QChartView" name="amp_ecg_axis" native="true">
         <property name="styleSheet">
          <string>background-color: white; border: 1px solid gray;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="amp_ppg_axis" native="true">
         <property name="styleSheet">
          <string>background-color: white; border: 1px solid gray;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="sleep_state_axis" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string>background-color: white; border: 1px solid gray;</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
          <property name="text">
           <string>Time (h)</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QChartView" name="ecg_axis" native="true">
       <property name="styleSheet">
        <string>background-color: white; border: 2px solid black;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QChartView" name="ppg_axis" native="true">
       <property name="styleSheet">
        <string>background-color: white; border: 1px solid black;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
